src/backend/distributed/sql/downgrades/citus--12.2-1--12.1-1.sql

```diff
@@ -21,3 +21,4 @@ DROP FUNCTION citus_internal.mark_object_distributed(
 DROP FUNCTION citus_internal.commit_management_command_2pc();
 
 ALTER TABLE pg_catalog.pg_dist_transaction DROP COLUMN outer_xid;
+REVOKE USAGE ON SCHEMA citus_internal FROM PUBLIC;
```

Comment:
Sidenote: Normally we'd want to revert the REVOKE on the functions here too (by running GRANT ON PUBLIC). But in this case it seems more of a hassle than it's worth, so it's fine to keep it like this.